packages/odyssey-react-mui/src/properties/odyssey-react-mui.properties

```diff
@@ -0,0 +1,7 @@
+field.error.screen.reader.text = Error
+field.label.optional.text = Optional
+severity.success = success
+severity.warning = warning
+severity.info = info
+severity.error = error
+toast.close.text = close
```

Comment:
We don't put this as `en` in the `translations/` folder?

Reply:
Negative as that is the "master" that you guys will own (english). Everything under `translations/` is owned by the translation provider and globalization core. Correct @nikhilvenkatraman-okta ?